solc/CommandLineInterface.cpp

```diff
@@ -765,7 +765,7 @@ void CommandLineInterface::compile()
 	catch (CompilerError const& _exception)
 	{
 		m_hasOutput = true;
-		formatter.printExceptionInformation(_exception, "Compiler error");
+		formatter.printExceptionInformation(_exception, Error::Type::CompilerError);
```

Comment:
Would this work?
```suggestion
		formatter.printExceptionInformation(_exception, _exception.type());
```
If it does, and `CompilerError` inherits from `Error` as I suspect, we could actually just remove this `catch` clause since the one below will do the same thing.

Reply:
Nope, it didn't work. Keeping it as it is for now.